.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macos-10.15, windows-2019]
-        sofa_branch: [master]
+        sofa_branch: [master, v22.12]
```

Comment:
Does it still make sense to rely on the `master` branch?

Reply:
You mean the workflow should be v22.12 and once release, master back ?

Reply:
My bad, I thought the PR was to be merged in the v22.12 branch. I did not see that the v22.12 branch was edited directly